### ISSUE TITLE
AMQ9638E: SSL communications error/2393 SSL unable to initialize - check SSL parms

### ISSUE THREAD

**Issue by abudavis** [ibm-messaging/mq-rfhutil#13]
We are using the latest rfhutilc (client) version from this git to connect to a remote QMGR and are facing errors during SSL handshake.

The setup consists of a remote qmgr which has a standard SSL setup configured (key.* files), the personal certificate has a certificate label "ibmwebspheremq<qmgrname>".

On the other side which is an rfhutilc.exe wherein it is pointed to a set of key.* files by clicking on "set connid".
The key.kdb contain a personal certificate with a certificate label  "ibmwebspheremqrfhutil" (also tried to use a label "ibmwebspheremq<qmgrname>").
SSL Cipher was set to "ANY_TLS12" on both sides of the channel.
Upon trying to connect from RFHUTIL client, we get the below error on MQ side and an error "2393 SSL unable to initialize - check SSL parms" on RFHUTIL client.

```
---MQ qmgr error log---
AMQ9638E: SSL communications error for channel ‘????’.
EXPLANATION:
An unexpected SSL communications error occurred for a channel, as reported in
the preceding messages. The channel is ‘????’; in some cases its name cannot be
determined and so is shown as ‘????’. The channel did not start.
-------------------------
```

The certificates on both sides are signed by the same CA and the signers are added to both sides. To verify if it is a QMGR problem, we setup a SDR-RCVR channel between this QMGR and another which seemed to work just fine.

What is it that we are missing here?

Reference article : https://developer.ibm.com/recipes/tutorials/connect-to-mq-on-cp4i-with-rfhutil

**Comment by lsabados** [comment]
I'd be interested in how to resolve this, as well.

**Comment by abudavis** [comment]
> I'd be interested in how to resolve this, as well.

I resolved the problem, it works now! Check the error logs on your MQ Client side (same machine from where you run your rfhutilc.exe). In my case, I had installed MQ client v9.2 and for that version, I simply had to add these 3 lines in the mqclient.ini as shown in the screenshot below. 

```
SSL:
      AllowTLSV13=TRUE
      OCSPCheckExtensions=NO 
      CDPCheckExtensions=NO
      OCSPAuthentication=OPTIONAL

```
![image](https://user-images.githubusercontent.com/33847625/107222715-8e593300-6a15-11eb-9d6e-85e3eda50286.png)
